storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -326,16 +330,60 @@
                                          (log-error t "Error when processing event")
                                          (exit-process! 20 "Error when processing an event")
                                          ))
+   :blob-update-timer (mk-timer :kill-fn (fn [t]
+                                           (log-error t "Error when processing event")
+                                           (exit-process! 20 "Error when processing a event")))
```

Comment:
Can we set `:timer-name` here in addition to the `:kill-fn`?
It might also be helpful for debugging to give a name here to the anonymous `fn`, as we have seen some mysterious hangs while logging.